### PR TITLE
Update README.md in Web development with examples of Full Stack Technologies #198 issue

### PR DESCRIPTION
**Pull request by dadigavasavipriya**
This pull request introduces comprehensive resource lists for various technology stacks used in web development. These stacks include LAMP, MEAN, Ruby on Rails, Python Django, Java, and ASP.NET. Each section contains curated links to tutorials, guides, and project ideas to help developers master full-stack development with these technologies.

Additions:

1.Introduction to Technology Stacks
Overview of full-stack development and the importance of mastering specific technology stacks.

2.Building Web Applications with LAMP Stack
Resources on Linux, Apache, MySQL, PHP.
Links to tutorials, roadmaps, and project examples.

3.Creating Web Apps with MEAN Stack
Resources on MongoDB, Express, Angular, Node.js.
Links to tutorials, roadmaps, and project examples.

4.Creating Web Applications with Ruby on Rails
Resources on Ruby, Rails, SQLite/MySQL/PostgreSQL, JavaScript.
Links to tutorials, roadmaps, and project examples.

5.Python Web Development using Django
Resources on Python, Django, React.js, MySQL.
Links to tutorials, roadmaps, and project examples.

6.Full-Stack Development with Java
Resources on Java, Spring Boot, React, MongoDB.
Links to tutorials, roadmaps, and project examples.

7.Developing Web Applications with ASP.NET
Resources on ASP.NET, C#, SQL Server.
Links to tutorials, roadmaps, and project examples.

8.Creating Web Apps with MERN Stack
Resources on MongoDB, Express, React, Node.js.
Links to tutorials, roadmaps, and project examples.

### CODE REVIEW

File: Web Development/readme.md [hunk 1]
```diff
@@ -43,6 +43,8 @@ This category is a curated collection of valuable web development resources cont
   - [Beginner level](#beginner-level-projects)
   - [Intermediate level](#intermediate-level-projects)
   - [Advanced level](#advance-level-projects)
+- [Full-stack with various technology stacks](#various-technolgy-stacks)
```
Comment (jfmartinz) [inline Web Development/readme.md:46]:
```suggestion
- [Technology Stacks](#technolgy-stacks)
```

File: Web Development/readme.md [hunk 2]
```diff
@@ -1658,3 +1660,529 @@ This section contains hundreds of coding challenges that test your knowledge of
         </td>
       </tr>  
   </table>
+
+### Various Technolgy Stacks
```
Comment (jfmartinz) [inline Web Development/readme.md:1664]:
```suggestion
### Technology Stacks
```

**Review by jfmartinz — CHANGES_REQUESTED** [review]
Also remove the description in each stacks in the table, only add description into the Technology Stacks and describe the content or about the resources included. Thanks

**Review by jfmartinz — APPROVED** [review]
Please remove these descriptions:
image
image

All the descriptions you added in the table. Only technology stacks should have a description, other tables below, please remove.


Another one is, please add those stacks in the table of contents, the same thing with what you see in the Projects section:

image

**Review by jfmartinz — APPROVED** [review]
Thanks @dadigavasavipriya

**Comment by dadigavasavipriya** [comment]
@jfmartinz Thank you for your feedback. I have removed the descriptions from the tables as requested, except for the Technology Stacks Additionally, I have added the Technology Stacks section to the table of contents, following the same format as the Projects section.
Thank you once again for assigning me this issue. Please let me know if there are any further adjustments needed.

**Comment by jfmartinz** [comment]
Merged your PR, thansk for your contribution

**Comment by dadigavasavipriya** [comment]
You're welcome, @jfmartinz! If there is any further changes required, please let me know.I'm glad to contribute to the project.

**Pull request merged**